src/App.js

```diff
@@ -43,14 +43,22 @@ export const App = props => {
             : ''
           : ''
 
-  const selectedTab = activeWidget ? activeWidget.name : props.widgets[0].name
+  var distinguisher = '-' + Math.random().toString(36).slice(2,5)
+  const retrieveSelectedTab = activeWidget ? activeWidget.name : props.widgets[0].name
+  let selectedTab
+
+  if (retrieveSelectedTab.includes('-')) {
+    selectedTab = retrieveSelectedTab
+  } else {
+    selectedTab = retrieveSelectedTab+distinguisher
+  }
+
   const tabLinks = props.widgets.map((widget, index) => {
-    return <TabLink to={widget.name} className='mr-4' key={`tabLink-${index}`}>{t(widget.name)}</TabLink>
+    return <TabLink to={widget.name+distinguisher} className='mr-4' key={`tabLink-${index}`}>{t(widget.name)}</TabLink>
   })
 
-
   const tabContents = props.widgets.map((widget, index) => {
-    return <TabContent for={widget.name} key={`tabContent-${index}`}>
+    return <TabContent for={widget.name+distinguisher} key={`tabContent-${index}`}>
```

Comment:
```suggestion
    return <TabContent for={widget.name + distinguisher} className={`mr-4 tabpanel-${widget.name.toLowerCase()}`} key={`tabContent-${index}`}>
```

Reply:
@mpolidori Now the ID will randomly be generated so, it will be difficult to styling but we can use a class instead of IDs.  
Also, it will be worth to add the ODDK issue on the PR description so that, everyone knows why we are implementing this.